login/index.html

```diff
@@ -26,17 +26,19 @@
     <div class="login_form">
       <form>
         <div>
-          <label>이메일</label>
-          <input type="email">
+          <label for="login-email">이메일</label>
+          <input id="login-email" class="login-email" type="email">
+          <div class="error_message"></div>
         </div>
         <div>
-          <label>비밀번호</label>
-          <input class="box_password" type="password">
+          <label for="login-password">비밀번호</label>
+          <input id="login-password" class="login-password" type="password">
+          <div class="error_message"></div>
         </div>
       </form>
     </div>
 
-    <a class="link_login" href="#">
+    <a id="login-button" class="link_login" href="#">
       로그인
     </a>
```

Comment:
a태그보다 form submit이 가능한 버튼으로 변경하는것이 좋을 것 같아요!
```suggestion
    <button type="submit" id="login-button" class="link_login">
      로그인
    </button>
```